src/main/java/lotto/domain/model/Lotto.java

```diff
@@ -58,17 +65,27 @@ private static List<Integer> generateNumberList() {
         return numberList;
     }
     
-    public Match match(Lotto winNumbers) {
-        return Match.fromLottoNumber(findMatchCount(winNumbers.numbers));
+    public Rank rankDecide(Lotto winNumbers) {
+        return Rank.fromLottoNumber(winNumbers.findMatchCount(this));
     }
     
-    public int findMatchCount(List<Integer> winNumbers) {
+    public int findMatchCount(Lotto winNumbers) {
```

Comment:
```suggestion
    private int findMatchCount(Lotto winNumbers) {
```
private으로 구현해도 되지 않을까?

Reply:
@javajigi 
처음에 이 구현할 때는 메서드의 접근제어자를 public으로 이용하면서 테스트 코드를 사용했습니다
그러다가 같은 객체 내 다른 메서드 안에서 사용하게 되어 private로 바꿔야 하게 되었습니다
그러나 private 로 하면 테스트코드를 쓸수가 없게 되어서 public으로 두었습니다.
일단 private 로 바꾸도록 할게요 👍